src/test/kotlin/org/rust/ide/typing/RustRawLiteralHashesDeleterTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.typing
+
+class RustRawLiteralHashesDeleterTest : RustTypingTestCaseBase() {
+    override val dataPath = "org/rust/ide/typing/rawHashes/fixtures"
+
+    // Swap file names, so we can reuse tests for inserter :-D
+    override fun beforeFileName(fileName: String): String {
+        return super.afterFileName(fileName)
+    }
+
+    override fun afterFileName(fileName: String): String {
+        return super.beforeFileName(fileName)
+    }
```

Comment:
Hm, and what is the problem? Usual tests boil down to [checkResultByFile](https://github.com/intellij-rust/intellij-rust/blob/4d4cc2dd0bf524cdc27f06fd7a7b53989023b14f/src/test/kotlin/org/rust/lang/RustTestCaseBase.kt#L44) which does uses `fileName` to configure fixtures, while the the inline style tests use `InlineFile`(https://github.com/intellij-rust/intellij-rust/blob/4d4cc2dd0bf524cdc27f06fd7a7b53989023b14f/src/test/kotlin/org/rust/lang/RustTestCaseBase.kt#L88) and configure files by text.  `//^` is used instead of `<caret>`, maybe `/*|*/` should be added as an alternative.